src/components/program-progress/ProgramProgressCircle.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { ProgramProgressContext } from './ProgramProgressContextProvider';
+
+const CircleSegment = ({
+  total, index, x, y, radius, degrees, strokeWidth, classList,
```

Comment:
can we declare constants on top, instead of taking them through arguments if they are never changing?